doc/man3/EVP_DigestSignInit.pod

```diff
@@ -166,6 +166,10 @@ external circumstances (see L<RAND(7)>), the operation will fail.
 The call to EVP_DigestSignFinal() internally finalizes a copy of the digest
 context. This means that calls to EVP_DigestSignUpdate() and
 EVP_DigestSignFinal() can be called later to digest and sign additional data.
+Note that not all providers allow for continuation, in case the selected
+provider does not allow to duplicate contexts EVP_DigestSignFinal() will
+finalize the digest context and attempting to process additional data via
+EVP_DigestSignUpdate() will result in an error.
```

Comment:
I see no code that does this, other than in your fake provider where you seem to add support for that.

Reply:
The fake provider emulates what the pkcs11-provider has to do, because it can't duplicate internal PKCS#11 operations in most cases.
It's the whole point of the test, again I do not understand what you are pointing at ...

Reply:
I think your latest commit addresses my concerns.